File: JavaScript/1064.js
var input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8');
var lines = input.split('\n');
let n = +lines.shift()
let par = 0;
for(let i=0;i<=6;i++){
    if(n > 0){
        par += 1
        i++
    }
}
console.log(`${i} valores positivos`)
console.log(`${par/n}`)

File: Python/1064.py
b = 0
aux = 0
media = 0
for i in range(0,6):
    a = float(input())
    if(a>0):
        b = b+1
        aux = a+aux
media = aux/b
print('%d valores positivos'%b)
print('%.1f'%media)